src/pathfinding_service/api.py

```diff
@@ -117,6 +118,13 @@ def __init__(self, debug_mode: bool, **kwargs: Any):
         self.debug_mode = debug_mode
 
     def post(self, token_network_address: str) -> Tuple[dict, int]:
+        graph_dir: Optional[str]
+        if self.debug_mode:
+            graph_dir = "/tmp/pfs_graphs"  # TODO: make this configurable
```

Comment:
Do we need to mount `/tmp` in the docer containers?

Reply:
I don't think we should do that. We should make that path configurable and do a bind mount for it. Until then we'll have fetch it from inside the container. Something like `docker run "cat /tmp/pfs_graphs/*.log"` should do it.

Reply:
We could also bind `/tmp/pfs_graphs`, but really think binding all of `/tmp` is a bad idea.

Reply:
Ah cool, I didn't know about `docker run "cat /tmp/pfs_graphs/*.log"`.